typescript.js

```diff
@@ -63,6 +63,27 @@ module.exports = {
             asyncArrow: 'always'
         }],
         'space-infix-ops': 'off',
-        '@typescript-eslint/space-infix-ops': 'error'
+        '@typescript-eslint/space-infix-ops': 'error',
+        /*
```

Comment:
These rules are more likely of interest to readers than the extensions rules. Therefore, they are higher priority, and I think it would be intuitive and an advantage to place them first.